frontend/src/components/eMIB/EmailPreview.jsx

```diff
@@ -1,30 +1,116 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
+import LOCALIZE from "../../text_resources";
 
 const styles = {
+  //buttons
   button: {
-    width: 180,
-    marginBottom: 10,
-    marginLeft: 20,
-    marginRight: 20,
-    textAlign: "center",
+    width: 155,
+    textAlign: "left",
     borderRadius: 4,
     padding: 6,
-    border: "2px solid #009fae"
+    border: "2px solid #00565E",
+    cursor: "pointer"
+  },
+  // 3x background colors
+  button_selected_background: {
+    backgroundColor: "#00565E"
+  },
+  button_read_background: {
+    backgroundColor: "#F5F5F5"
+  },
+  button_unread_background: {
+    backgroundColor: "white"
+  },
+  // 2x text colors
+  button_selected_text: {
+    color: "#D3FCFF"
+  },
+  button_unselected_text: {
+    color: "black"
+  },
+  //li
+  li: {
+    listStyleType: "none"
+  },
+  //subject line
+  subject_selected: {
+    color: "white"
+  },
+  subject_unselected: {
+    color: "#00565E"
+  },
+  subject_read: {
+    fontWeight: "normal"
+  },
+  subject_unread: {
+    fontWeight: "bold"
   }
 };
 
 class EmailPreview extends Component {
   static propTypes = {
     email: PropTypes.object.isRequired,
-    clickFunction: PropTypes.func.isRequired
+    clickFunction: PropTypes.func.isRequired,
```

Comment:
This could be called something a bit more specific, like `selectEmail`

Reply:
Fixed in b168ca5